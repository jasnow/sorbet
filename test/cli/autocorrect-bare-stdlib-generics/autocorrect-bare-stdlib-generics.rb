# typed: strict

extend T::Sig

sig { params(stuff: T::Array).void }
def bare_array(stuff); end

sig { params(stuff: T::Hash).void }
def bare_hash(stuff); end

sig { params(stuff: T::Set).void }
def bare_set(stuff); end

sig { params(stuff: T::Range).void }
def bare_range(stuff); end

sig { params(stuff: T::Enumerable).void }
def bare_enumerable(stuff); end

sig { params(stuff: T::Enumerator).void }
def bare_enumerator(stuff); end
