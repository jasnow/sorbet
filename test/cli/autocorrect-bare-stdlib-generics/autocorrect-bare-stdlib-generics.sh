tmp="$(mktemp)"
file="test/cli/autocorrect-bare-stdlib-generics/autocorrect-bare-stdlib-generics.rb"
cp "$file" "$tmp"
main/sorbet --silence-dev-message -a "$tmp"
diff "$file" "$tmp"
rm "$tmp"
