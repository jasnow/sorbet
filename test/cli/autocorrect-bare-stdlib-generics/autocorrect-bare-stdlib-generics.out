5c5
< sig { params(stuff: T::Array).void }
---
> sig { params(stuff: T::Array[T.untyped]).void }
8c8
< sig { params(stuff: T::Hash).void }
---
> sig { params(stuff: T::Hash[T.untyped, T.untyped]).void }
11c11
< sig { params(stuff: T::Set).void }
---
> sig { params(stuff: T::Set[T.untyped]).void }
14c14
< sig { params(stuff: T::Range).void }
---
> sig { params(stuff: T::Range[T.untyped]).void }
17c17
< sig { params(stuff: T::Enumerable).void }
---
> sig { params(stuff: T::Enumerable[T.untyped]).void }
20c20
< sig { params(stuff: T::Enumerator).void }
---
> sig { params(stuff: T::Enumerator[T.untyped]).void }
